biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.spec.tsx

```diff
@@ -0,0 +1,22 @@
+import { cleanup, getByText, render, wait } from '@testing-library/react';
+import React from 'react';
+import InputPage from './input-pagee';
+
+describe('Info page', () => {
+  afterEach(() => {
+    delete global['fetch'];
+    cleanup();
+  });
+
+  it('should render successfully', async () => {
+    const message = 'my message';
+    global['fetch'] = jest.fn().mockResolvedValueOnce({
+      json: () => ({
+        message,
+      }),
+    });
+
+    const { baseElement } = render(<InputPage />);
+    await wait(() => getByText(baseElement, message));
+  });
+});
```

Comment:
nit: test for the number of times the fetch function(site calls for the message) is called.

Reply:
How could I go about doing this?

Reply:
@RolandNaijuka I am going to merge for now so that we can all work off of master, but we can include this change in another PR?

Reply:
@Lev-Stambler  There is a function in jest that tests that amount of times a fetch is called ```expect(fetch).toHaveBeenCalledTimes(1);```
Check this [link](https://medium.com/@rishabhsrao/mocking-and-testing-fetch-with-jest-c4d670e2e167) for more information on how to use it